docs/pages/admin-guides/management/admin/logging.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Logger configuration
+description: Logger configuration for Teleport instance
+---
+
+## Configuration
```

Comment:
```suggestion
```
I don't think there's a need for this H2 here, since there are no other headings, and the title already presents the topic of the guide.